src/main/java/frc/robot/commands/Autos/AutonomousAll.java

```diff
@@ -11,11 +11,14 @@
 import frc.robot.commands.drivetrain.ForwardDistance;
 import frc.robot.commands.intake.Direction;
 import frc.robot.commands.intake.DoubleIntake;
+import frc.robot.commands.newClimber.ArmPneumaticTippingMid;
+import frc.robot.commands.newClimber.ArmPneumaticTippingTraverse;
 import frc.robot.commands.oldClimber.ArmPneumaticTipping;
-import frc.robot.commands.oldClimber.ArmPneumaticTipping.ArmTipState;
+import frc.robot.subsystems.NewClimber;
+import frc.robot.subsystems.OldClimber;
 import frc.robot.subsystems.Drivetrain;
 import frc.robot.subsystems.Intake;
-import frc.robot.subsystems.OldClimber;
+import frc.robot.ArmTipState;
 
 public class AutonomousAll extends SequentialCommandGroup {
```

Comment:
I am not sold on having one auto that deals with both climbers. I think I would have made two different autos.